transient/transient.py

```diff
@@ -228,13 +228,18 @@ def __qemu_added_args(self) -> List[str]:
                 ssh_bin_name=self.config.ssh_bin_name,
             )
 
+            if self.config.net_driver is None:
+                net_driver = "virtio-net-pci"
+            else:
+                net_driver = self.config.net_driver
```

Comment:
I think this should be set in by using the 'missing' field in the configuration.py

Reply:
Updated